datastore/prometheus.go

```diff
@@ -0,0 +1,318 @@
+package datastore
+
+import (
+	"context"
+	"net/http"
+	"strconv"
+	"sync"
+
+	"github.com/ddosify/alaz/log"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+const (
+	projectLabel   = "acorn.io/project-name"
+	appLabel       = "acorn.io/app-public-name"
+	containerLabel = "acorn.io/container-name"
+	accountIdLabel = "acorn.io/account-id"
+)
+
+type PrometheusExporter struct {
+	ctx context.Context
+	reg *prometheus.Registry
+
+	latencyHistogram *prometheus.HistogramVec
+	statusCounter    *prometheus.CounterVec
+
+	podCache *eventCache
+	svcCache *eventCache
```

Comment:
These caches are used to store information about known pods and services in the cluster.

Reply:
It looks like these caches are never expired/emptied. Is that something we want to worry about?

Reply:
Fixed. Handling deletion events now